include/nbl/asset/utils/CHLSLCompiler.h

```diff
@@ -55,6 +55,8 @@ class NBL_API2 CHLSLCompiler final : public IShaderCompiler
 		// when Nabla is used as a lib
 		nbl::asset::hlsl::impl::DXC* m_dxcCompilerTypes;
 
+		IShaderCompiler::CIncludeFinder* m_defaultIncludeFinder;
+
```

Comment:
its already provided by compiler options struct

Reply:
actually its already in `IShaderCompiler::m_defaultIncludeFinder`, and `CHLSLCompiler` inherits from that

Reply:
REMOVE THE MEMBER!